src/AutoRest.CSharp/Mgmt/AutoRest/MgmtConfiguration.cs

```diff
@@ -46,7 +46,7 @@ public class MgmtDebugConfiguration
                 SkipCodeGen = Configuration.DeserializeBoolean(skipCodeGen, false);
                 GenerateReport = Configuration.DeserializeBoolean(generateReport, true);
                 ReportOnly = Configuration.DeserializeBoolean(reportOnly, false);
-                ReportFormat = reportFormat?.GetString() ?? "yaml";
+                ReportFormat = Configuration.IsValidJsonElement(reportFormat) ? reportFormat?.GetString() ?? "yaml" : "yaml";
```

Comment:
Shall we throw an exception if the format is invalid?

Reply:
I do not think so - this is not a code gen related feature, it is only generating the reports. Not everyone is using it therefore making it throw any exception does not make sense.
BTW here IsValidJsonElement is not checking if the format is invalid as it is implied. It is checking if the kind of this element is null or invalid, this is not always caused by an invalid input. When omitting the format, we will get a `default` JsonElement for this thing, and default always has a `invalid` kind